dodo.py

```diff
@@ -558,6 +558,48 @@ def task_pytest():
             ],
         }
 
+def task_creds():
+    '''
+    check for valid aws credentials
+    '''
+    def uptodate():
+        try:
+            call('aws sts get-caller-identity')
+            return True
+        except CalledProcessError:
+            return False
+    return {
+        'task_dep': [
+            'check',
+        ],
+        'actions': [
+            'echo "missing valid AWS credentials"',
+            'false',
+        ],
+        'uptodate': [
+            uptodate
+        ],
+    }
+
+def task_domain():
```

Comment:
task_domain should not delete domain unless specified.

Reply:
impossible to achieve as it is currently configured